### DISCUSSION TITLE
必填项的样式对其Antd

### DISCUSSION THREAD

**Discussion by Alex-xd** [alibaba/formily#3643] [RFC]
如下图所示，希望能在FormLayout中加入该项requiredMark配置，支持这种新的选填项样式。目前我们设计师都是按照Antd的设计规范在做，这个地方看了下目前除了改formily源码否则很难通过其他手段定制，望考虑支持下谢谢

![image](https://user-images.githubusercontent.com/11256006/209533449-693b9899-7b67-48f1-aa23-e5482f15a79a.png)

https://4x-ant-design.antgroup.com/components/form-cn/#components-form-demo-required-mark

**Comment by Alex-xd** [comment]
改一下FormItem的这个renderLabelText是不是就可以了？  https://github.com/alibaba/formily/blob/formily_next/packages/antd/src/form-item/index.tsx#L225

**Comment by zxq08** [comment]
哈喽大佬 请问下后来怎么解决了。感觉antd-v5和antd不同步这块确实是个隐患，是否有方案可以共用一套呢
